server/models/games-models.js

```diff
@@ -68,3 +68,32 @@ exports.selectReviews = () => {
       return err;
     });
 };
+exports.selectCommentsByReviewId = async (review_id) => {
+  if (isNaN(+review_id)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid input"
+    });
+  }
+
+  const reviewCheck = await connection.query(
+    `SELECT * FROM comments
+     WHERE review_id = $1`,
+    [review_id]
+  );
+  if (reviewCheck.rowCount === 0) {
+    return Promise.reject({
+      status: 404,
+      msg: "Resource not found"
+    });
+  }
+
+  const comments = await connection.query(
+    `SELECT * FROM comments
+     WHERE review_id = $1`,
+    [review_id]
+  );
```

Comment:
You should be able to remove this second query just have an else to return the rows if the rowCount is not 0